www/guides/react.md

```diff
@@ -0,0 +1,371 @@
+---
+layout: layouts/guide.njk
+permalink: '/guides/react/'
+title: The Snowpack Guide to Getting Started with React
+description: 'Get started with this in-depth tutorial on how to build React applications and websites with Snowpack. '
+date: 2020-10-01
+---
+
+Image: ideally a nice header image showing the before or after, or some illustration with elements of the Snowpack logo, React, and symbols related to speed, joy.
+
+Snowpack is a great fit for [React](https://reactjs.org/) projects of any size. It's easy to get started and can scale to projects containing thousands of components and pages without any impact on development speed. Unlike traditional React application tooling, Snowpack saves you from getting bogged down with complex bundler setups and configuration files.
+
+In this guide, you'll go from an empty directory to a fully configured Snowpack project with support for React and several other useful developer tools. In the process, you'll learn:
+
+- How to set up Snowpack
+- How to get started with React in a Snowpack project
+- Importing CSS, images and other assets
+- Enabling [Fast Refresh](https://reactnative.dev/docs/fast-refresh) mode for React
+- Connecting your favorite developer tools to Snowpack
+
+> 💡 Tip: if you want to jump to the end to see a full featured React setup, the [Create Snowpack App React template](https://github.com/pikapkg/snowpack/tree/master/create-snowpack-app/app-template-react) comes with everything you'll learn in this guide plus other useful tools.
+>
+> ```bash
+> npx create-snowpack-project react-snowpack-template --template @snowpack/project-template-react
```

Comment:
Is there a reason this template name is different from the snippet right below? I don’t want super beginners to be confused something isn’t working if they copied this one by mistake

Reply:
Oh I’m a dummy this is the finished project. WDYT about moving this snippet to the very end, and only linking to the last heading here? That way the only snippets they see are copy-safe (but can still jump to the end if they want)